libs/components/modals/testing/src/confirm/confirm-harness.ts

```diff
@@ -0,0 +1,100 @@
+import { SkyComponentHarness } from '@skyux/core/testing';
+import { SkyConfirmType } from '@skyux/modals';
+
+/**
+ * Harness for interacting with a confirm component in tests.
+ * @internal
+ */
+export class SkyConfirmHarness extends SkyComponentHarness {
+  public static hostSelector = 'sky-confirm';
+
+  #getMessageEl = this.locatorFor('.sky-confirm-message');
+  #getBodyEl = this.locatorForOptional('.sky-confirm-body');
+  #getButtonsEl = this.locatorForAll('.sky-confirm-buttons .sky-btn');
+
+  /**
+   * Gets the message of the confirm component.
+   */
+  public async getMessageText(): Promise<string> {
+    return await (await this.#getMessageEl()).text();
+  }
+
+  /**
+   * Gets the body of the confirm component.
+   */
+  public async getBodyText(): Promise<string | undefined> {
+    const bodyEl = await this.#getBodyEl();
+
+    if (bodyEl) {
+      return await bodyEl.text();
+    }
+    return undefined;
+  }
+
+  /**
+   * Gets the type of the confirm component.
+   */
+  public async getType(): Promise<SkyConfirmType | undefined> {
```

Comment:
Can the return type be `Promise<SkyConfirmType>` since all code paths return a value?